contracts/src/v0.1/L1EndpointVRF.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+import "./L1EndpointBase.sol";
+import "./interfaces/IVRFCoordinator.sol";
+
+abstract contract L1EndpointVRF is L1EndpointBase {
+    IVRFCoordinator VRFCOORDINATOR;
+
+    error OnlyVRFCoordinatorCanFulfill(address have, address want);
+    event RandomWordRequested(uint256 requestId, address sender);
+    event RandomWordFulfilled(
+        uint256 requestId,
+        uint256 l2RequestId,
+        address sender,
+        uint256 callbackGasLimit,
+        uint256[] randomWords
+    );
+
+    constructor(address _vrfCoordinator) {
```

Comment:
@KelvinThai Is there a reason why we use `RequestResponseConsumerBase` for `L1EdnpointRequestResponse` but not using `VRFConsumerBase` in here?

Reply:
I think we might not need the the `RequestResponseConsumerBase`.